src/app/backend/resource/endpoint/list.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 The Kubernetes Dashboard Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"github.com/kubernetes/dashboard/src/app/backend/api"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+type EndpointList struct {
+	ListMeta api.ListMeta `json:"listMeta"`
+	// List of endpoints
+	Endpoints []Endpoint `json:"endpoints"`
+}
+
+// ToEndpointList converts array of api events to endpoint List structure
+func ToEndpointList(endpoints []v1.Endpoints, nonCriticalErrors []error) EndpointList {
```

Comment:
Is it used anywhere outside the package? Maybe we can keep it internal (change to `toEndpointList`).